web/src/containers/activity/EntryDetails.test.js

```diff
@@ -1,22 +1,37 @@
 import { shallow } from 'enzyme';
 import React from 'react';
 
-import { plain as EntryDetails, mapStateToProps } from './EntryDetails';
+import { removeActivity } from '../../actions/editActivity';
+
+import {
+  plain as EntryDetails,
+  mapStateToProps,
+  mapDispatchToProps
+} from './EntryDetails';
 
 describe('the (Activity) EntryDetails component', () => {
   const props = {
     activityIndex: 2,
     activityKey: 'activity key',
     fundingSource: 'money pit',
-    name: 'activity name'
+    name: 'activity name',
+    remove: jest.fn()
   };
 
+  beforeEach(() => {
+    props.remove.mockClear();
+  });
+
   test('renders correctly with the modal closed', () => {
     const component = shallow(<EntryDetails {...props} />);
     expect(component).toMatchSnapshot();
```

Comment:
I think it'd be useful to have a separate test here where `activityIndex` is zero. Just a simple snapshot test, to make sure the delete button isn't rendered in that case.